docs/APIOverview.mdx

```diff
@@ -3,15 +3,15 @@ id: api-overview
 title: API Overview
 ---
 
-This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.org/docs/en/get-started) and [how Casbin works](https://casbin.org/docs/en/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
+This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.io/docs/get-started) and [how Casbin works](https://casbin.io/docs/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
 
 ## Enforce API
 
-Let's start at the Enforce APIs of Casbin. We will load a RBAC model from `model.conf`, and load policies from `policy.csv`. You can learn the Model syntax [here](https://casbin.org/docs/en/syntax-for-models), and we won't talk about it in this tutorial. We assume that you can understand the config files given below:
+Let's start at the Enforce APIs of Casbin. We will load a RBAC model from `model.conf`, and load policies from `policy.csv`. You can learn the Model syntax [here](https://casbin.io/docs/syntax-for-models), and we won't talk about it in this tutorial. We assume that you can understand the config files given below:
 
 `model.conf`
 
-```ini
+```
```

Comment:
Why remove this?

Reply:
This repo is base on https://github.com/casbin/casbin-website/pull/265#issuecomment-1152577649, it has old documentation.
In fact, there is no `ini` in the [original document](https://github.com/casbin/casbin-website/blob/14a4ceba447b8d5ff996319677da79fd2a76c54c/docs/APIOverview.md?plain=1#L14). But it really should have `ini`.
The reason for this problem is my commit base one #2 , and I didn't notice this kind of problem.(also the `bash`,`Javascript`).
I will fix it in the next commit.